interop-tests/src/test/scala/akka/grpc/scaladsl/PowerApiSpec.scala

```diff
@@ -32,16 +43,30 @@ class PowerApiSpec(backend: String)
     with AnyWordSpecLike
     with Matchers
     with ScalaFutures
-    with Directives {
+    with Directives
+    with BeforeAndAfter {
 
   override implicit val patienceConfig = PatienceConfig(5.seconds, Span(10, org.scalatest.time.Millis))
 
+  val numberOfRepliesOverStreams = 5
+
   val server =
-    Http().newServerAt("localhost", 0).bind(GreeterServicePowerApiHandler(new PowerGreeterServiceImpl())).futureValue
+    Http()
+      .newServerAt("localhost", 0)
+      .bind(GreeterServicePowerApiHandler(new PowerGreeterServiceImpl(numberOfRepliesOverStreams)))
+      .futureValue
+
+  var client: GreeterServiceClient = _
+
+  after {
+    if (client != null && !client.closed.isCompleted) {
+      client.close().futureValue
+    }
+  }
```

Comment:
The implementation using the netty client will add some noise on the logs if the client is not closed nicely.